codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/RulesBasedAuthSchemeResolverGenerator.kt

```diff
@@ -150,6 +142,26 @@ class RulesBasedAuthSchemeResolverGenerator {
         }
     }
 
```

Comment:
Below lines add auth option customization for SSO and SSO OIDC to rules based auth scheme resolver as well. Addition of function is just to reduce duplication above.